Unix/wsman/wsmanparser.c

```diff
@@ -2929,6 +2935,54 @@ int WS_ParseFaultBody(
                     if (XML_Expect(xml, &e, XML_END, PAL_T('s'), PAL_T("FaultDetail")) != 0)
                         RETURN(-1);
                 }
+                else if (Tcscmp(e.data.data, ZT("OMI_Error")) == 0)
+                {
+                    int i;
+                    for (i=0; i<e.attrsSize; ++i)
```

Comment:
there is an XML_GetAttr or something similar so save having to iterate through it